src/software/simulated_tests/er_force_simulator.py

```diff
@@ -0,0 +1,217 @@
+from subprocess import Popen
+
+from proto.geometry_pb2 import Angle, AngularVelocity, Point, Vector
+from proto.messages_robocup_ssl_wrapper_pb2 import SSL_WrapperPacket
+from proto.robot_status_msg_pb2 import RobotStatus
+from proto.sensor_msg_pb2 import SensorProto
+from proto.vision_pb2 import BallState, RobotState
+from proto.world_pb2 import SimulatorTick, WorldState
+
+from software.networking.threaded_unix_listener import ThreadedUnixListener
+from software.networking.threaded_unix_sender import ThreadedUnixSender
+
+WORLD_STATE_PATH = "/world_state"
```

Comment:
I don't like the multiple sources of truth on these constants for py vs c++, seems like we should be able to pass them in from python to C++